Paragraphs about author commitment (#391)
ropensci/dev_guide: softwarereview_author.Rmd

## Planning a Submission (or a Pre-Submission Enquiry)

> -   Do you expect to maintain your package for at least 2 years, or to be able to identify a new maintainer?

Author: Depending on the duration we find reasonable, like the software-review PR

- Please consider the best time in your package's development to submit.  Your package should be sufficiently mature so that reviewers are able to review all essential aspects, but keep in mind that review may result in major changes.
    - We strongly suggest submitting your package for review _before_ publishing on CRAN or submitting a software paper describing the package to a journal. Review feedback may result in major improvements and updates to your package, including renaming and breaking changes to functions.
    -   Do not submit your package for review while it or an associated manuscript is also under review at another venue, as this may result in conflicting requests for changes.

> - Please also consider the time and effort needed to respond to reviews (for your package's best!): think about your availability or those of your collaborators in the next weeks and months following a submission. We tend to be flexible with deadlines and submissions can be put on hold but we want to avoid wasting volunteer reviewers' time.

Reviewer: ```suggestion
- Please also consider the time and effort needed to respond to reviews: think about your availability or that of your collaborators in the next weeks and months following a submission. We tend to be flexible with deadlines, and submissions can be put on hold but we want to avoid wasting volunteer reviewers' time.
```

-   An [editor](#editors) will review your submission within 5 business days and respond with next steps. The editor may assign the package to reviewers, request that the package be updated to meet minimal criteria before review, or reject the package due to lack of fit or overlap.
-   If your package meets minimal criteria, the editor will assign  1-3 reviewers. They will be asked to provide reviews as comments on your issue within 3 weeks.
-   We ask that you respond to reviewers' comments within 2 weeks of the last-submitted review, but you may make updates to your package or respond at any time. Your response should include a link to the updated [NEWS.md](#news) of your package. Here is [an author response example](https://github.com/ropensci/software-review/issues/160#issuecomment-355043656).  We encourage ongoing conversations between authors and reviewers. See the [reviewing guide](#reviewerguide) for more details.

> -   If you are no longer able to maintain your package and respond to reviews, please strive to find a successor for that role so as not to waste reviewers' time. You can also post in the rOpenSci slack workspace.

Reviewer: ```suggestion
-   Please notify us immediately if you are no longer able to maintain your package or to respond to reviews. You will then be expected to either retract a submission, or to find alternative package maintainers. You can also dsicuss maitenance issues in the rOpenSci slack workspace.
```
